src/deployer.py

```diff
@@ -138,16 +139,19 @@ def _get_next_job_port() -> int:
         return 8000
 
     def save_job_secrets(self,
-                            job_name: str,
-                            job_version: str,
-                            job_secrets: JobSecrets,
-                            ):
-        raise NotImplementedError("managing secrets is not supported on local docker")
+                         job_name: str,
+                         job_version: str,
+                         job_secrets: JobSecrets,
+                         ):
+        self._secrets_store[f'{job_name}.{job_version}'] = job_secrets
```

Comment:
It would be nice if this logged a warning, since this is a non-ideal hopefully temporary fix.